src/rules/require-sql-template.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+const assert = require('assert');
+const flora = require('flora-sql-parser');
+const get = require('lodash.get');
+
+/**
+ * Instances.
+ */
+
+const parser = new flora.Parser();
+
+/**
+ * Check if `literal` is an SQL query.
+ */
+
+function isSqlQuery(literal) {
+  if (!literal) {
+    return false;
+  }
+
+  try {
+    parser.parse(literal);
+  } catch (err) {
```

Comment:
Can't remove `err` since it results in a `SyntaxError` (not allowed by the language).

Also, `parser.parse()` throws a custom and weird `SyntaxError` which is not an instance of `Error` and is hard to handle.

In any case, for this specific purpose, we should just go with `return false` because we don't really wan't errors to propagate to the linting script.